### DISCUSSION TITLE
Chart is not displayed when a short time frame is specified In Netdata Cloud

### DISCUSSION THREAD

**Discussion by soichisumi** [netdata/netdata#13399] [Q&A] [answered]
If I select Last 10min, the Chart not be displayed, but Last 2hour, its displayed.
The graph shows that the data for the last 10min exists, so it does not seem that data is missing.

![image](https://user-images.githubusercontent.com/30210641/179639035-4112b3b9-478e-4235-a829-efc1be077e27.png)

![image](https://user-images.githubusercontent.com/30210641/179639018-da91f515-fb12-4613-b424-f99860f8d557.png)

**Comment by MrZammler** [comment]
Hi @soichisumi ! What agent version are you using?

**Comment by soichisumi** [comment]
Hi, @MrZammler. I'm using `netdata v1.35.0-191-nightly`

> `$ netdata -version`
> `netdata v1.35.0-191-nightly`

**Reply by MrZammler** [reply]
Thanks!

Can you please do the following? Try to replicate the problem, and then please share the output from `access.log`. The requests for data should appear there, along with any likely error from the agent.

There should be some lines starting with `2022-07-19 10:36:09: 0: 9670 '[ACLK]:1' 'DATA'`.

**Reply by soichisumi** [reply]
thank you for your advice, i'll check today!

**Reply by soichisumi** [reply]
Hi, @MrZammler. I did more investigation

After reproducing the problem, I checked netdata/access.log.

There may be no errors, and there don't seem to be many relevant errors in error.log either.

[accesslog.txt](https://github.com/netdata/netdata/files/9161056/accesslog.txt)

Error.log;
```
2022-07-22 00:50:55: tc-qos-helper.sh: WARNING: FireQOS is not installed on this system. Use FireQOS to apply traffic QoS and expose the class names to netdata. Check https://github.com/netdata/netdata/tree/master/collectors/tc.plugin#tcplugin
2022-07-22 00:50:55: tc-qos-helper.sh: WARNING: Cannot find file '/usr/lib/netdata/conf.d/tc-qos-helper.conf'.
2022-07-22 00:50:55: tc-qos-helper.sh: WARNING: Cannot find file '/etc/netdata/tc-qos-helper.conf'.
```


I also found that during the error, the request to node was failing.

![image](https://user-images.githubusercontent.com/30210641/180284235-27a0b993-ea3b-4c04-851c-c554bd9bb01f.png)

**Reply by soichisumi** [reply]
I have also discussed this matter on Discord and was directed to this Issue.

Issue : https://github.com/netdata/netdata-cloud/issues/489
Netdata Discord thread: https://discord.com/channels/847502280503590932/998747614331801721

**Comment by soichisumi** [comment, marked as answer]
Issue(Answer) : https://github.com/netdata/netdata-cloud/issues/489
Netdata Discord thread(For reference): https://discord.com/channels/847502280503590932/998747614331801721
